plugins-compat-tester/src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -463,7 +463,7 @@ private TestExecutionResult testPluginAgainst(MavenCoordinates coreCoordinates,
             while (entries.hasMoreElements()) {
                 JarEntry entry = entries.nextElement();
                 String name = entry.getName();
-                Matcher m = Pattern.compile("WEB-INF/lib/jenkins-core-([0-9.]+(?:-[0-9.]+)?(?:-SNAPSHOT)?)[.]jar").matcher(name);
+                Matcher m = Pattern.compile("WEB-INF/lib/jenkins-core-([0-9.]+(?:-[0-9.]+)?(?:-(alpha|beta|RC|rc)-[0-9.]+)?(?:-SNAPSHOT)?)[.]jar").matcher(name);
```

Comment:
@jtnord IMHO SNAPSHOT should not be case insensitive because is an special case in maven, agree on the other cases, will change that